lib/test/reporters.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { ObjectDefineProperties } = primordials;
+
+let dot;
+let spec;
+let tap;
+
+ObjectDefineProperties(module.exports, {
```

Comment:
are we sure that cjs-module-lexer will be able to pick up this pattern as named exports?

Reply:
it is a pretty common pattern in node's built-in modules, so it probably does:
https://github.com/nodejs/node/blob/9dbb1623ffd6e06b5df0c74beaf3f4ee0a17c8e5/lib/crypto.js#L340
https://github.com/nodejs/node/blob/9dbb1623ffd6e06b5df0c74beaf3f4ee0a17c8e5/lib/buffer.js#L1392
https://github.com/nodejs/node/blob/9dbb1623ffd6e06b5df0c74beaf3f4ee0a17c8e5/lib/dns.js#L345
https://github.com/nodejs/node/blob/9dbb1623ffd6e06b5df0c74beaf3f4ee0a17c8e5/lib/os.js#L399
https://github.com/nodejs/node/blob/9dbb1623ffd6e06b5df0c74beaf3f4ee0a17c8e5/lib/zlib.js#L926

Reply:
It probably doesn’t but we don’t use `cjs-module-lexer` for core modules, because core modules are not CJS (or at least, not regular CJS) and the use of named imports for core modules predates the introduction of `cjs-module-lexer`.